docs/additional-functionality/rapids-shuffle.md

```diff
@@ -340,6 +340,52 @@ non-standard location.
 With the RAPIDS Shuffle Manager configured, the setting `spark.rapids.shuffle.enabled` (default on)
 can be used to enable or disable the usage of RAPIDS Shuffle Manager during your application.
 
+#### Databricks
+
+Please make sure you follow the plugin's [Getting Started](../get-started/getting-started-databricks.md)
```

Comment:
Nit: Ideally we should use "RAPIDS Accelerator" when referring to the plugin in user documentation.  In this case, I think it's extraneous, especially since we link to the doc.
```suggestion
Please make sure you follow the [Getting Started](../get-started/getting-started-databricks.md)
```